modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/UpdateObjectEntryObjectActionExecutorImpl.java

```diff
@@ -116,6 +121,11 @@ private void _execute(
 				new ObjectEntry() {
 					{
 						properties = values;
+						status = new Status() {
+							{
+								code = serviceBuilderObjectEntry.getStatus();
+							}
+						};
```

Comment:
What do you thing to inline the `serviceBuilderObjectEntry`? Something like this:
 ```java
new ObjectEntry() {
	{
		properties = values;

		setStatus(
			() -> {
				com.liferay.object.model.ObjectEntry
					serviceBuilderObjectEntry =
						_objectEntryService.getObjectEntry(
							primaryKey);

				return new Status() {
					{
						code =
							serviceBuilderObjectEntry.
								getStatus();
					}
				};
			});
	}
});
 ```

Reply:
We can do that